infra/build-services.md

```diff
@@ -0,0 +1,36 @@
+# Choice of Build Pipelines
+
+Currently, our builds run on [Azure Dev Ops Pipelines](https://dev.azure.com/responsibleai/fairlearn/_build) (hereafter ADO).
+This was an easy default choice for a project growing out of work from Microsoft Research.
+We were already familiar with ADO, and when we need more resources, linking to subscriptions owned by the AzureML team is straightforward.
+However, using ADO could create a false perception as to the degree of control Microsoft exerts over Fairlearn.
+We want Fairlearn to be a good open source citizen.
+
+## An Alternative: GitHub Actions
+
+GitHub offers a similar feature to ADO pipelines, named [GitHub Actions](https://github.com/features/actions).
+We already use an Action to auto-assign reviewers, and have recently created a proof-of-concept Actions pipeline to run our current notebooks against our PyPI package (to catch breaking changes).
+
+Being part of GitHub, switching to GitHub Actions will likely improve the perception of Fairlearn as a open source project (which happens to have major contributions from Microsoft).
+One major barrier to this move is the lack of templating in defining Actions pipelines.
+In ADO, we make heavy use of templating to eliminate duplicate code and have testing pipline which run our tests against multiple platforms and Python versions.
+Not only does Actions lack templating, but there is no clear roadmap as to when they will be supported.
+
+## Long Term Plan: Move to GitHub Actions
+
+In the long term, moving our build and release pipelines to GitHub Actions appears to be the logical choice.
+We will be a better open source citizen while keeping the same functionality.
+
+## Short Term Plan: Organising the move
```

Comment:
Can we settle on American English for everything related to Fairlearn?

Reply:
I trust the new draft meets with your approval.

Reply:
or you can go canadian and accept both ;)